src/components/ButtonLink.jsx

```diff
@@ -7,9 +8,21 @@ export default function ButtonLink({
   href,
   external = false,
   newTab = false,
+  isHashLink = false,
   linkProps,
   ...rest
 }) {
+  if (isHashLink) {
+    return (
+      <HashLink
+        style={{ textDecoration: 'unset' }}
+        to={href}
+        {...linkProps}
```

Comment:
It doesn't seem to hurt anything, but HashLink is probably not expecting these props.